seatunnel-dist/src/test/java/org/apache/seatunnel/api/connector/MarkdownHeaderTest.java

```diff
@@ -41,6 +41,43 @@ public static void setup() {
         docsDirectorys.add(Paths.get("..", "docs", "zh"));
     }
 
+    @Test
+    public void testFileNameVerify() {
+        // Verify that the file names in the English and Chinese directories are the same.
+        List<String> enFileName =
+                fileName(docsDirectorys.get(0)).stream()
+                        .map(path -> path.replace("/en/", "/"))
+                        .collect(Collectors.toList());
+        List<String> zhFileName =
+                fileName(docsDirectorys.get(1)).stream()
+                        .map(path -> path.replace("/zh/", "/"))
+                        .collect(Collectors.toList());
+
+        long equalCount = enFileName.stream().filter(zhFileName::contains).count();
+        long equalIgnoreCaseCount =
+                enFileName.stream()
+                        .map(String::toLowerCase)
+                        .filter(
+                                name ->
+                                        zhFileName.stream()
+                                                .map(String::toLowerCase)
+                                                .anyMatch(name::equals))
+                        .count();
+
+        Assertions.assertEquals(equalCount, equalIgnoreCaseCount);
```

Comment:
Can we throw exception of the chinese doc name can not be found in english version when doc not match?